### DISCUSSION TITLE
[bug] filters on iteration breaks when field is renamed

### DISCUSSION THREAD

**Discussion by seangarner** [community/community#12645] [Projects and Issues]
I noticed that you can rename the `iteration` field in settings now.

![image](https://user-images.githubusercontent.com/1780230/157678950-41038700-9d84-441f-a16b-d229f38f944b.png)

But when I renamed it to `sprint`, filters referencing it broke.

![image](https://user-images.githubusercontent.com/1780230/157679046-e1f318d9-ccf8-4103-be69-d3e0ea42bceb.png)

It's a minor gripe, but it'd be handy if the filters were automatically updated as part of renaming the field.

Also related, but I can't reproduce.  When I first renamed it and noticed the broken filter I figured it just needed renaming so went to update it, but the new name didn't work either until I refreshed my browser.

**Comment by volkantash** [comment]
Az önce gördük. Biri hemen bu bozukluğu onarmalı. (We just saw it. Someone should fix this malfunction immediately.)
👨‍💻👨‍🔧👨‍🚒🧯
